src/napari_micromanager/_mda_handler.py

```diff
@@ -8,10 +8,11 @@
 
 import napari
 import zarr
+from pymmcore_plus.mda.handlers._util import get_full_sequence_axes
```

Comment:
never import private stuff from other packages,  even if it's your package.  I know it's annoying.  but it just always comes back to bite you.  I'll update this one